src/extlibs/hrvo/simulator.cpp

```diff
@@ -377,29 +377,25 @@ void HRVOSimulator::visualize(unsigned int robot_id) const
 {
     // TODO (#2499): Create a new HRVO visualization proto and uncomment/update
     // LOG(VISUALIZE)
```

Comment:
does this resolve 2499?

Reply:
Since we're still technically not visualizing, and the LOG(VISUALIZE) call in this function is still commented out until we get the python side of it done, I decided to keep the TODO here. Though it can also be removed.

Reply:
gotcha, yeah we can uncomment the LOG(VIZ) and remove the TODO

Reply:
Sounds good! I've uncommented the log viz and removed the TODO